ui/admin/app/components/form/credential-library/vault-ssh-certificate/index.hbs

```diff
@@ -0,0 +1,292 @@
+        <F.Error as |E|>
+          {{#each @model.errors.key_bits as |error|}}
+            <E.Message>{{error.message}}</E.Message>
+          {{/each}}
+        </F.Error>
+      {{/if}}
+    </Hds::Form::TextInput::Field>
+  {{/if}}
+
+  <Hds::Form::TextInput::Field
+    @isOptional={{true}}
+    @value={{@model.ttl}}
+    @isInvalid={{@model.errors.ttl}}
+    @type='text'
+    name='ttl'
+    disabled={{form.disabled}}
+    {{on 'input' (set-from-event @model 'ttl')}}
+    as |F|
+  >
+    <F.Label>{{t 'resources.credential-library.form.ttl.label'}}</F.Label>
+    <F.HelperText>{{t
+        'resources.credential-library.form.ttl.help'
+      }}</F.HelperText>
+    {{#if @model.errors.ttl}}
+      <F.Error as |E|>
+        {{#each @model.errors.ttl as |error|}}
+          <E.Message>{{error.message}}</E.Message>
+        {{/each}}
+      </F.Error>
+    {{/if}}
+  </Hds::Form::TextInput::Field>
+
+  <Hds::Form::TextInput::Field
+    @isOptional={{true}}
+    @value={{@model.key_id}}
+    @isInvalid={{@model.errors.key_id}}
+    @type='text'
+    name='key_id'
+    disabled={{form.disabled}}
+    {{on 'input' (set-from-event @model 'key_id')}}
+    as |F|
+  >
+    <F.Label>{{t 'resources.credential-library.form.key_id.label'}}</F.Label>
+    <F.HelperText>{{t
+        'resources.credential-library.form.key_id.help'
+      }}</F.HelperText>
+    {{#if @model.errors.key_id}}
+      <F.Error as |E|>
+        {{#each @model.errors.key_id as |error|}}
+          <E.Message>{{error.message}}</E.Message>
+        {{/each}}
+      </F.Error>
+    {{/if}}
+  </Hds::Form::TextInput::Field>
+
+  <Form::Field::KeyValueList
+    @layout='horizontal'
+    @isOptional={{true}}
+    @name='critical_options'
+    @options={{@model.critical_options}}
+    @addOption={{fn this.addOption 'critical_options'}}
+    @removeOptionByIndex={{fn this.removeOptionByIndex 'critical_options'}}
+    @disabled={{form.disabled}}
+    as |M|
+  >
+    <M.Legend>{{t
+        'resources.credential-library.form.critical_options.label'
+      }}</M.Legend>
+    <M.HelperText>{{t
+        'resources.credential-library.form.critical_options.help'
+      }}</M.HelperText>
+    {{#if @model.errors.critical_options}}
+      <M.Error as |E|>
+        {{#each @model.errors.critical_options as |error|}}
+          <E.Message>{{error.message}}</E.Message>
+        {{/each}}
+      </M.Error>
+    {{/if}}
+  </Form::Field::KeyValueList>
+
+  <Form::Field::KeyValueList
+    @layout='horizontal'
+    @isOptional={{true}}
+    @name='extensions'
+    @options={{@model.extensions}}
+    @addOption={{fn this.addOption 'extensions'}}
+    @removeOptionByIndex={{fn this.removeOptionByIndex 'extensions'}}
+    @disabled={{form.disabled}}
+    as |M|
+  >
+    <M.Legend>{{t
+        'resources.credential-library.form.extensions.label'
+      }}</M.Legend>
+    <M.HelperText>{{t
+        'resources.credential-library.form.extensions.help'
+      }}</M.HelperText>
```

Comment:
Cleanup the formatting here?

Reply:
Good catch, I had to manually format this as it seems both `prettier` and `ember-template-lint` don't see this as a problem...